iree/base/dynamic_library_test_library.cc

```diff
@@ -21,7 +21,7 @@
 #if defined(_WIN32)
 #define IREE_SYM_EXPORT __declspec(dllexport)
 #else
-#define IREE_SYM_EXPORT
+#define IREE_SYM_EXPORT __attribute__ ((visibility ("default")))
```

Comment:
clang-format doesn't want a space here

```suggestion
#define IREE_SYM_EXPORT __attribute__((visibility ("default")))
```